test/e2e/nightwatch/tests/index.js

```diff
@@ -1,11 +1,19 @@
-var dragAndDrop = require('../../../../lib/index').codeForSelectors;
+var dragAndDrop = require('../../../../lib/index');
 
 module.exports = {
-  'Drag and Drop with Nightwatch': function(browser) {
+  'Drag and Drop with Nightwatch using CSS selectors': function(browser) {
     browser
       .url('https://kuniwak.github.io/html-dnd/test.html')
       .assert.containsText('#result', 'NG')
-      .execute(dragAndDrop, ['#draggable', '#droppable'])
+      .execute(dragAndDrop.codeForSelectors, ['#draggable', '#droppable'])
+      .assert.containsText('#result', 'OK')
+      .end();
+  },
+  'Drag and Drop with Nightwatch using XPath': function(browser) {
```

Comment:
I think this test case should split to another file.

In my understanding, the official document of Nightwatch.js said that we should call `browser.end()` only once on a file:

> Remember always to call the .end() method when you want to close your test, in order for the Selenium session to be properly closed.
>
>  http://nightwatchjs.org/guide#writing-tests

But `browser.end()` is unexpectedly called twice in the file.
So It will cause a test failure like:

```
Running:  Drag and Drop with Nightwatch using CSS selectors
 ✔ Testing if element <#result> contains text: "NG".
 ✔ Testing if element <#result> contains text: "OK".
No assertions ran.

Running:  Drag and Drop with Nightwatch using XPath
 ✖ Testing if element <#result> contains text: "NG". Element could not be located.  - expected "NG" but got: null
```